### ISSUE TITLE
VGG with CUB200 without pre-trained

### ISSUE THREAD

**Issue by iamzifei** [bentrevett/pytorch-image-classification#6]
Hi this is a really great repo with detailed explanation. I'm trying to using the CUB200 dataset from the 5-ResNet example with 4-VGG net, however what I find is with `model.load_state_dict(pretrained_model.state_dict())`, I'm able to get the expected accuracy in the first few epoches. However, without loading pretrained parames, the accuracy in the first few epoches is 0.x%, which looks really wrong.

I've put a modified notebook here https://colab.research.google.com/drive/1Tu5hPH51y6Up9pIIJiUO-9aHXz4A-MJK?usp=sharing

Not sure what I missing and the reason behind the issue
